pandas/core/indexes/base.py

```diff
@@ -5557,6 +5557,30 @@ def _should_fallback_to_positional(self) -> bool:
         missing : np.ndarray[np.intp]
             An indexer into the target of the values not found.
             These correspond to the -1 in the indexer array.
+
+        Examples
+        --------
+        >>> index = pd.Index(['c', 'b', 'a', 'b', 'b'])
+        >>> index.get_indexer_non_unique(['b', 'b'])
+        (array([1, 3, 4, 1, 3, 4]), array([], dtype=int64))
+
+        In the example below there are no matched values. For this reason,
+        the returned ``indexer`` contains only integers equal to -1.
+        It demonstrates no index at these positions that match the corresponding
+        ``target`` values. The mask [0, 1, 2] in the return value shows that the first,
+        second, and third elements are missing.
```

Comment:
Just suggesting a slight rewording
```suggestion
        It demonstrates that there's no match between the index and the ``target``
        values at these positions. The mask [0, 1, 2] in the return value shows that
        the first, second, and third elements are missing.
```